wolfcrypt/src/port/Renesas/renesas_rx64_hw_util.c

```diff
@@ -0,0 +1,96 @@
+/* renesas_rx64_hw_util.c
+ *
+ * Copyright (C) 2020-2022 Johnson Controls International Plc
+ *
```

Comment:
I am not sure if we can accept this copy right. I leave this to a senior engineer review.